package.json

```diff
@@ -25,10 +25,10 @@
     "prepare": "yarn build",
     "start": "yarn storybook",
     "storybook": "start-storybook -p 3000 -s public/development",
-    "test": "jest --detectOpenHandles",
+    "test": "cross-env NODE_OPTIONS='--max-old-space-size=8096' jest --detectOpenHandles",
     "test:a11y": "jest --testPathPattern='src\\/([^\\s]+)\\/([a-zA-Z0-9\\s_\\\\.\\-\\(\\):])+(\\.test\\.a11y\\.)+(js|jsx)$'",
-    "test:watch": "jest --watchAll --runInBand --verbose",
-    "test:update": "jest --updateSnapshot",
+    "test:watch": "cross-env NODE_OPTIONS='--max-old-space-size=8096' jest  --watch --verbose",
+    "test:updateSnapshot": "cross-env NODE_OPTIONS='--max-old-space-size=8096' jest  --updateSnapshot",
```

Comment:
could we put this back to `test:update`?

Reply:
@scottdickerson

Reply:
I pushed a commit to resolve this.